### PR TITLE
created a ssiservice component

### LINKED ISSUE #661: Create component for SSI Service Prereq
since we're repeating the same thing over and over, I suggest making this a component and just embedding it in these guides. That way if we ever need to make any changes to this, we will only need to do so in one place

image

### PR DESCRIPTION
**Pull request by ShavonHarris-Dev**
## What type of PR is this? (check all applicable)

- [x] ♻️ Refactor
- [ ] ✨ New Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR [adds/removes/fixes/replaces] this [feature/bug/etc]. 

## Related Tickets & Documents

Resolves #661

## Added code snippets?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: site/src/components/SSIComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const SSIService = () => {
```
Comment (angiejones) [inline site/src/components/SSIComponent.jsx:3]: can we rename this (and the file) to SSISPrereq?

File: site/docs/ssi/create-did.mdx [hunk 2]
```diff
@@ -4,14 +4,17 @@ title: Create a DID
...
+
 # Create a Decentralized Identifier (DID)
 
 In the world of SSI, [DIDs](/docs/web5/learn/decentralized-identifiers) are used to identify any subject (e.g., a person, organization, thing, data model, abstract entity, etc.). In this tutorial we'll show you how to create a DID using the [SSI Service](https://github.com/TBD54566975/ssi-service). 
 
 :::info
 ## Prerequisites
 
-Follow guide to [Clone & Run SSI Service](run-ssi-service).
+<SSIComponent />
```
Comment (angiejones) [inline site/docs/ssi/create-did.mdx:17]: here are all the places we should this
https://developer.tbd.website/search?q=Clone%20%26%20Run%20SSI%20Service

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/docs/ssi/create-did.mdx [hunk 3]
```diff
@@ -4,14 +4,17 @@ title: Create a DID
...
 
+import SSIComponent from '@site/src/components/SSIComponent'
+
+
 # Create a Decentralized Identifier (DID)
 
 In the world of SSI, [DIDs](/docs/web5/learn/decentralized-identifiers) are used to identify any subject (e.g., a person, organization, thing, data model, abstract entity, etc.). In this tutorial we'll show you how to create a DID using the [SSI Service](https://github.com/TBD54566975/ssi-service). 
 
 :::info
 ## Prerequisites
```
Comment (angiejones) [inline site/docs/ssi/create-did.mdx:15]: can remove this header, because it's getting printed twice
image

**Comment by ShavonHarris-Dev** [comment]
> @ShavonHarris-Dev Hey, just checking in, do you need any help on this issue? we can do a call in discord if you've gotten stuck on anything

Hi @EbonyLouis I was actually not well and was out. Will get to this today/ tomorrow. If thats ok? Thanks for checking in.

**Comment by EbonyLouis** [comment]
@ShavonHarris-Dev no rush, just checking up on you. I hope you feel better

**Comment by ShavonHarris-Dev** [comment]
> @ShavonHarris-Dev no rush, just checking up on you. I hope you feel better

Hey @EbonyLouis ! Im finally back on my feet. I went in and made the component more dynamic so that it could be used all over. Can you let me know if this is what you were going for? Want to make sure i'm on the right path! Thanks :)

**Comment by blackgirlbytes** [comment]
Hey @ShavonHarris-Dev I sent you a quick text, but our team decided to be prioritize this issue, so I'm gonna close this pull request.  I appreciate your hard work immensely!

**Pull request closed without merging**